docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -134,6 +102,37 @@ If you want to provision a new volume or restore the existing one, create on-dem
       apiGroup: snapshot.storage.k8s.io
   ```
 
+  </details>
+  <details>
+  <summary label="AKS">
+  AKS
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.17.
+
+### Steps
+
+  1. [Install the CSI driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md).
+  2. [Create a volume snapshot](https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/deploy/example/snapshot).
+
+  </details>
+  
+  <details>
+  <summary label="GKE">
+  GKE
+  </summary>
+
+### Prerequisites
+
+  The minimum supported Kubernetes version is 1.14.
```

Comment:
can we not have 1.17 here too for sake of uniformity ?

Reply:
Now the recommended k8s version is 1.21 and soon will be 1.22. I would rather use that infact because its tested in these version only.

Reply:
solved: removed the prerequisites, as agreed with @a-thaler